### ISSUE TITLE
Undefined error

### ISSUE THREAD

**Issue by tomadmiraal** [davicotico/Menu-Editor#4]
Your old menu editor didn't add an empty children array when there were not children.

Can you add a check for the children-variable to be present so the items are backwards compatible?

**Comment by davicotico** [comment]
Yes, that improvement will be included in an upcoming release. Thanks for your feedback.

**Comment by davicotico** [comment]
Fixed in v1.2.0
